wire-library/wire-runtime-swift/build.gradle

```diff
@@ -19,8 +19,28 @@ dependencies {
   wire project(':wire-compiler')
 }
 
+task generateSwiftProtos(type: JavaExec) {
```

Comment:
so, I hate this. There's got to be a better way. Anyone with more knowledge, please help me level up my gradle-foo